tests/end2end/test_user_inspection.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Test inspecting user."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from flask import url_for
+from flask_babel import gettext as _
+
+
+def test_superuser_can_inspect_user(superuser, user, testapp):
+    """Inspect user details as superuser."""
+    # Goes to homepage.
+    res = testapp.get('/')
+    # Fills out login form.
+    form = res.forms['loginForm']
+    form['username'] = superuser.email
+    form['password'] = 'myPrecious'
+    # Submits.
+    res = form.submit().follow()
+    # Clicks Users button.
+    res = res.click(_('Users'))
```

Comment:
I kinda like the `url_for` version (e.g. like the changes in `test_permission_editing.py`), what are your feelings?

Reply:
I like both..